app/services/piv_cac_service.rb

```diff
@@ -3,6 +3,8 @@
 
 module PivCacService
   class << self
+    RANDOM_HOSTNAME_BYTES = 2
```

Comment:
I'm open to argument but I'd vote to make this bigger, maybe 4? Every time somebody gets confused and hits Esc or something, they increase their probability of a collision. The longer the random string, the uglier the URL, but the lower the probability of a user-hostile collision.

Reply:
This is confusing. Two bytes == four hex digits in the hostname.